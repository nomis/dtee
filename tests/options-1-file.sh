#!/bin/bash
set -vx
. "$(dirname "$0")"/util/common.sh

run_test_once \
	"-o" "stdout_append" \
	"-e" "stderr_append" \
	"-c" "combined_append" \
	"-O" "stdout_overwrite" \
	"-E" "stderr_overwrite" \
	"-C" "combined_overwrite" \
	"--debug-options"
RET=$?

echo RET $RET
if [ $RET -eq 0 ]; then
	exit 0
fi
exit 1
