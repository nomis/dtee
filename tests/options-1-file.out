combined-append=combined_append
combined-overwrite=combined_overwrite
debug-options=1
err-append=stderr_append
err-overwrite=stderr_overwrite
out-append=stdout_append
out-overwrite=stdout_overwrite
